/**:
  ros__parameters:

    ns: [ "center" ]

    ### input topic info, (yaw,pitch,roll in degree/ x,y,z in meter)
    in_clouds:
      center:
        pose_yaw: 0.0
        pose_pitch: 0.0
        pose_roll: 0.0
        pose_x: 0.787
        pose_y: 0.0
        pose_z: 1.733 # lowest: 1.733 (0.09)

    out_cloud:
      frame_id: "base_link"
      period_ms: 100
      # rate: 10 # Hz
